LEAF_Nexus/sources/Employee.php

```diff
@@ -298,14 +298,18 @@ public function getSummary($empUID)
 
     /**
      * Get positions associated with an employee
-     * @param int $positionID
-     * @return array
+     *
+     * @param int |
+     * @return int
+     * Created at: 1/18/2023, 2:16:45 PM (America/New_York)
      */
-    public function getPositions($empUID) 
+    public function getPositions(int | string $empUID): int | array | bool
     {
```

Comment:
I don't think you want spaces before or after |. Notice in the PHPDoc section that string is not associated with the @param, nor is array and bool with the @return